src/docfx.website.themes/pdf.default/gulpfile.js

```diff
@@ -30,4 +30,11 @@ gulp.task('concat', function () {
   ;
 });
 
+gulp.task('copy:font', function () {
+    return gulp.src(vendor.font.src, { cwd: vendor.font.cwd })
+        .pipe(copy('./fonts/'));
+});
+
+gulp.task('copy', ['copy:font']);
```

Comment:
What about add `copy` step into `default` task? If the font is required, I think the default step should include it.

Reply:
OK, I changed it to:

`gulp.task('default', gulp.parallel('concat', 'copy:font'));`